rust/impl/ca/ca/commands_aggregator.rs

```diff
@@ -30,6 +33,46 @@ pub( crate ) mod private
     pub exit_code_on_error : Option< i32 >,
     /// Commands.
     pub commands : std::collections::HashMap< String, Command >,
+    /// Commands context.
+    pub context : Option< Context >,
+  }
+
+  #[ derive( Debug, Clone ) ]
+  /// Basic context which contains commands hashmap
+  pub struct Context
+  {
+    inner : Rc< RefCell< dyn std::any::Any > >
```

Comment:
I do not understand why the std::any::Any is used. Please, give me link to such code in the notan.